x-pack/plugins/cases/public/components/user_actions/comment/alert.tsx

```diff
@@ -104,3 +94,55 @@ export const createAlertAttachmentUserActionBuilder = ({
     ];
   },
 });
+
+const getFirstItem = (items?: string | string[] | null): string | null => {
+  if (items == null) {
```

Comment:
as items can also be undefined it can be clearer if you validate for both null and undefined.

Reply:
just for the sake of fun, this function can be reduced to this:

```  
   return Array.isArray(items) ? items[0] : items || null;
```

Reply:
I think is better to use `??` instead of `||`.

Reply:
> as items can also be undefined it can be clearer if you validate for both null and undefined.

The equality operator `==` validates for both `null` and `undefined`, right?

Reply:
yes it does. that's why this is a nitpick, because it can be clearer but no need to change it if you don't want to.

Reply:
FWIW, I agree with @academo here and prefer strict comparison with `===`, for example as done in this helper function https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/common/types/guards.ts.

Reply:
Yes, if you want to check only for `null`. Although you can catch a nasty bug if the value is `undefined` for some reason. If you want to check for both `null` and `undefined` do you prefer `value !== null || value !== undefined`? I found it a bit redundant tbh.

Reply:
Yes, I'd use `value !== undefined && value !== null`